.settings/org.eclipse.jdt.core.prefs

```diff
@@ -0,0 +1,4 @@
+eclipse.preferences.version=1
+org.eclipse.jdt.core.compiler.codegen.targetPlatform=1.8
+org.eclipse.jdt.core.compiler.compliance=1.8
+org.eclipse.jdt.core.compiler.source=1.8
```

Comment:
Just to check, are we building for java 1.8? :D

Reply:
Yeah, exactly! :)